provisioning/cloudformation/templates/workshop-jenkins.json

```diff
@@ -325,46 +334,8 @@
                 "service nginx restart\n",
                 "# Install apache benchmark - Easy Capacity Testing\n",
                 "apt-get install -y apache2-utils\n",
-                "# Install nodejs for config-rules-status\n",
-                "curl -sL https://deb.nodesource.com/setup_6.x | bash -\n",
                 "apt-get install -y nodejs\n",
-                "npm install --global serverless@0.5.5\n",
-                "npm install --global gulp-cli\n",
-                "pip install awscli\n",
-                "# Install config-rules-status\n",
-                "# Go-go gadget Rube Goldberg machine\n",
-                "chmod 777 /opt\n",
-                "pushd /opt\n",
```

Comment:
this was moved out of running via serverless.js in metadata with hardcoded keys, to new rake task using ruby sdk for configservice